plugins/inputs/opentelemetry_listener/grpc_services.go

```diff
@@ -0,0 +1,72 @@
+package opentelemetry_listener
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb-observability/otel2influx"
+	otlpcollectorlogs "github.com/influxdata/influxdb-observability/otlp/collector/logs/v1"
+	otlpcollectormetrics "github.com/influxdata/influxdb-observability/otlp/collector/metrics/v1"
+	otlpcollectortrace "github.com/influxdata/influxdb-observability/otlp/collector/trace/v1"
+)
+
+type grpcServiceBase struct {
+	influxConverter *otel2influx.OpenTelemetryToInfluxConverter
+	influxWriter    *writer
```

Comment:
For me this would be better, but it's a matter of taste:
```suggestion
	converter *otel2influx.OpenTelemetryToInfluxConverter
	writer    *influxWriter
```